libs/ett/auth/data-access-api/src/lib/oidc/auth-oidc-http.service.ts

```diff
@@ -41,6 +41,21 @@ export interface AuthOidcLoginResponse {
   readonly url: string;
 }
 
+export interface AuthOidcLogoutResponse {
+  readonly success: boolean;
+}
+
+export interface UserProfile {
+  readonly id: string;
+  readonly name: string;
+  readonly company?: string;
```

Comment:
question: Can `company` be `null`? Then the type should be:
```suggestion
  readonly company: string | null;
```